src/packages/sdk/src/logger.ts

```diff
@@ -22,3 +23,6 @@ export function info(message: any, ...optionalParams: any[]) {
 export function error(message: any, ...optionalParams: any[]) {
   console.error(`${tags.error} ${message}`, ...optionalParams)
 }
+export function query(message: any, ...optionalParams: any[]) {
```

Comment:
Looks like it's not used, will it be used / useful later?

Reply:
It could be and it also keeps the loggers API consistent

Reply:
Yes